packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js

```diff
@@ -4828,9 +4828,117 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
   {
     component: "Search input",
     description: "Removed component",
-    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5506 ",
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5506",
     details: "The search input component was removed in favor of the text input group component.",
     repo: "HTML/CSS",
     fixedWithCodeMod: false
-  }
+  },
+  {
+    component: "Quickstarts",
+    description: "Made compatible with V5",
+    pullRequestURL: ["https://github.com/patternfly/patternfly-quickstarts/pull/250", "https://github.com/patternfly/patternfly-quickstarts/pull/252"],
+    repo: "Quickstarts",
```

Comment:
```suggestion
    repo: "patternfly-quickstarts",
```

Reply:
quickstarts is the only package that doesn't have `patternfly` before it in the name

Reply:
we could have it say patternfly/quickstarts, but we aren't doing that for the other repos... so I don't know if we should do it here like patternfly/react-topology? that's technically the topology repo.
And it'd be patternfly/patternfly for the HTML/CSS repo, for example.

Reply:
oh no worries then, I just assumed based off of the repo name in GH that it was patternfly-quickstarts. If that's wrong I'd stick with what you have then!